src/main/java/com/group7/ecommerce/controller/api/ShipInfoController.java

```diff
@@ -0,0 +1,110 @@
+package com.group7.ecommerce.controller.api;
+
+import com.group7.ecommerce.dto.request.ShipInfoRequest;
+import com.group7.ecommerce.dto.response.ApiResponse;
+import com.group7.ecommerce.dto.response.ShipInfoResponse;
+import com.group7.ecommerce.service.ShipInfoService;
+import com.group7.ecommerce.utils.CustomUserDetails;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Locale;
+
+@RestController
+@RequestMapping("/api/customer/ship-info")
```

Comment:
```suggestion
@RequestMapping("/api/ship-info")
```

chị thắc mắc sao phải thêm chữ customer vậy

Reply:
theo tên role nhóm em để chị ạ